pkg/storaged/crypto-keyslots.jsx

```diff
@@ -303,6 +303,17 @@ function ensure_package_installed(steps, progress, package_name) {
                                 return install_missing_packages(data, status_callback(progress));
                         }
                     });
+            })
+            .catch(error => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18698-20230424-134533-fc33405c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)